timeserio/keras/utils.py

```diff
@@ -22,7 +23,8 @@ def iterlayers(model: "keras.engine.Layer") -> Iterable["keras.engine.Layer"]:
         yield model
 
 
-def seed_random():
+@contextmanager
```

Comment:
the problem is I am using the `seed_random` explicitely in one test, and it wouldn't be importable from conftest (otherwise you're right, a fixture would provide a context manager etc)